tests/integrations/test_dbm.py

```diff
@@ -24,17 +23,68 @@ def setup_trace_payload(self):
             ]
         elif self.library_name == "dotnet":
             self.requests = [
-                weblog.get("/dbm", params={"integration": "npgsql"}, timeout=20),
                 weblog.get("/dbm", params={"integration": "mysql"}),
+                weblog.get("/dbm", params={"integration": "npgsql"}, timeout=20),
                 weblog.get("/dbm", params={"integration": "sqlclient"}),
             ]
 
-    def test_trace_payload(self):
+    def setup_trace_payload_disabled(self):
+        self.weblog_trace_payload()
+
+    @scenarios.integrations_service
+    def setup_trace_payload_service(self):
+        self.weblog_trace_payload()
+
+    @scenarios.integrations
+    def setup_trace_payload_full(self):
+        self.weblog_trace_payload()
+
+    def test_trace_payload_disabled(self):
+        for r in self.requests:
+            assert r.status_code == 200, f"Request: {r.request.url} wasn't successful."
+            for _, trace in interfaces.library.get_traces(request=r):
+                db_span = None
+                for span in trace:
+                    if span.get("resource") == "SELECT version()" or span.get("resource") == "SELECT @@version":
+                        db_span = span
+                        break
+
+                assert (
+                    db_span is not None
+                ), "No DB span with expected resource 'SELECT version()' nor 'SELECT @@version' found."
```

Comment:
nit: you could extract this to a helper function that gets the `db_span`, as it's duplicated between your three tests?